### ISSUE TITLE
[Content Gap] loading config from file

### ISSUE THREAD

**Issue by sryza** [dagster-io/dagster#4629]
## What content is missing

Would be good to mention this on our config overview page or something.

## Anything else?

---

Writer's Guide: [Docs README](https://github.com/dagster-io/dagster/tree/master/docs)

---
#### Message from the maintainers:
Are you looking for the same documentation content? Give it a :thumbsup:. We factor engagement into prioritization.

**Comment by erinkcochran87** [comment]
@sryza - Is this something other than what's documented [here](https://docs.dagster.io/concepts/configuration/config-schema#command-line), specifically using the CLI to load a file?

**Comment by sryza** [comment]
Yes - there are utility methods we provide: https://docs.dagster.io/_apidocs/utilities#dagster.config_from_files.

E.g. here's something a user might want to do:
```
my_configured_airbyte_resources = airbyte_resources.configured(config_from_files(["my_airbyte_config.yaml"]))
```

(I haven't exactly tested that to make sure it works)
